web/src/billing/sponsorship.ts

```diff
@@ -93,9 +95,8 @@ export function initialize(): void {
 
     function update_discount_details(): void {
         const selected_org_type =
-            $<HTMLSelectElement>("select#organization-type")
-                .find(":selected")
-                .attr("data-string-value") ?? "";
+            the($<HTMLSelectElement>("select#organization-type")).selectedOptions[0]?.dataset
+                .stringValue ?? "";
```

Comment:
Can we avoid using `dataset` with things named with dashes? It breaks greppability.

I suppose a valid fix would be to change the HTML attribute `data-string-value` to `data-stringValue`?

Reply:
Nope, `data-stringValue` corresponds to `dataset.stringvalue`, because HTML attributes are case-insensitive. This is just how the web platform works, and it’s better for everyone to learn and embrace how it works, rather than having each developer fight it over their own different minuscule surface-level details.

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dataset

We already use `dataset.codeLanguage`, `dataset.tabKey`, `dataset.sendStamp`, `dataset.tippyPlacement`, `dataset.tooltipTemplateId` (and I’m pretty sure I’m only responsible for one of these).

The better solution would be to avoid having a `data-string-value` at all. What’s currently the `data-string-value` should just be the `value`—which means that would be what gets passed in the API, rather than the current meaningless integers. I didn’t do that here because it’s a bigger change, but I think it’d be fine to just change this non-public API.